src/Components/SingleArticle.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import api from "./api";
+
+const SingleArticle = () => {
+  const { article_id } = useParams();
```

Comment:
Great use of useParams to ensure this page works if a user goes directly to the article 👍